docs/source/user-manual/blocks.md

```diff
@@ -54,32 +56,193 @@ Now with your empty block available, you can select its type in one of two ways.
 
 ## Description Block
 
-A description block provides a blank textarea to write plain text.
+These blocks allow users to add text content to a page. They can be configured to display different fonts, sizes, and colors, and they can also include links and formatting options like bold and italic.
```

Comment:
No, the description block is plain text and is in sync with the field "description", which should be plain text for SEO.

Reply:
should I replace it wtih this -> `The description block is a simple block that can be used to add plain text` or the previous description works -> `A description block provides a blank textarea to write plain text.` Thanks!

Reply:
```suggestion
A description block accepts unformatted plain text.
When displayed, it appears as the description in the page, and for search engine optimization in HTML meta tags as `<meta name="description" content="DESCRIPTION_TEXT" data-react-helmet="true">` and `<meta property="og:description" content="DESCRIPTION_TEXT" data-react-helmet="true">`.
```